ejb-txn-remote-call/server/pom.xml

```diff
@@ -0,0 +1,95 @@
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.wildfly.quickstarts</groupId>
+        <artifactId>ejb-txn-remote-call</artifactId>
+        <version>20.0.0.Beta1-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>ejb-txn-remote-call-server</artifactId>
+    <packaging>war</packaging>
+    <name>Quickstart: ejb-txn-remote-call-server</name>
+    <description>The project is the application to be deployed on the second server to receive the call</description>
+
+    <properties>
+        <version.narayana>5.9.5.Final</version.narayana>
+    </properties>
+
+    <dependencies>
+        <!-- Import the WS/JAX-RS -->
+        <dependency>
+            <groupId>org.jboss.spec.javax.ws.rs</groupId>
+            <artifactId>jboss-jaxrs-api_2.1_spec</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <!-- Import the EJB API -->
+        <dependency>
+            <groupId>org.jboss.spec.javax.ejb</groupId>
+            <artifactId>jboss-ejb-api_3.2_spec</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <!-- Import the Common Annotations API (JSR-250) -->
+        <dependency>
+            <groupId>org.jboss.spec.javax.annotation</groupId>
+            <artifactId>jboss-annotations-api_1.3_spec</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <!-- Import the JTA API -->
+        <dependency>
+            <groupId>org.jboss.spec.javax.transaction</groupId>
+            <artifactId>jboss-transaction-api_1.3_spec</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <!-- Import the JPA/Hibernate API -->
+        <dependency>
+            <groupId>jakarta.persistence</groupId>
+            <artifactId>jakarta.persistence-api</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <!-- Import logging by JBoss -->
+        <dependency>
+            <groupId>org.jboss.logging</groupId>
+            <artifactId>jboss-logging</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <!-- for Mock XAResource recovery we need the dependency to Narayana -->
+        <dependency>
+            <groupId>org.jboss.narayana.jta</groupId>
+            <artifactId>jta</artifactId>
+            <version>${version.narayana}</version>
```

Comment:
This should be avoided if possible, if this artifact is in general useful for apps then we should add it to jakartaee8 BOM at wildfly/boms, along other transaction artifacts. I didn't find it on wildfly dependency management, but perhaps we can add it to the BOM through version ref? Please see https://github.com/wildfly/boms/blob/master/jakartaee8/pom.xml#L515

Reply:
Ok, I see. I haven't realized this. I think I can use the `narayana jts` artifact instead. That one wraps whole the Narayana functionality. The `narayana jts` should be part of the BOM, I expect.